packages/framework/src/Component/Elasticsearch/ElasticsearchStructureManager.php

```diff
@@ -87,7 +87,7 @@ protected function getDefinition(int $domainId, string $index): array
         if (!is_file($file)) {
             throw new ElasticsearchStructureException(
                 sprintf(
-                    'Definition file %d.json, for domain: %1$d, not found in definition folder "%s". Please make sure that for each domain exists a definition json file with name of a domain ID',
+                    'Definition file %d.json, for domain ID %1$d, not found in definition folder "%s". Please make sure that for each domain exists a definition json file named by the corresponding domain ID',
```

Comment:
?? `for domain with ID ` ??
Maybe you can use PHP_EOL as separator after the first sentence 
But OK.

Reply:
PHP_EOL added, the first suggestion not :grin: